src/main/java/com/cedrus/patpetillo/springkafkapingpong/config/DatabaseConfig.java

```diff
@@ -0,0 +1,34 @@
+package com.cedrus.patpetillo.springkafkapingpong.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+
+import javax.sql.DataSource;
+
+@Configuration
+@ComponentScan("com.cedrus.patpetillo.springkafkapingpong")
+@PropertySource("classpath:database.properties")
+public class DatabaseConfig {
+    @Autowired
+    Environment environment;
+
+    private final String URL = "url";
```

Comment:
Hardcoded private finals are generally static.  They can't be anything but this, so why not define them at the Class level?